src/core/lib/channel/connected_channel.cc

```diff
@@ -609,6 +625,12 @@ class ClientStream : public Orphanable {
   struct Closed {};
   struct SendMessageToTransport {};
 
+  enum class ServerInitialMetadataState : uint8_t {
+    kNotReceived,
```

Comment:
Can you please add comments indicating exactly what each of these enum values represents?  It's not immediately clear from the names whether "received" means from the transport and "set" means from the surface, or vice-versa.

Reply:
done!